csharp/ql/src/Security Features/CWE-639/InsecureDirectObjectReference.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Insecure Direct Object Reference
+ * @description Using user input to control which object is modified without
+ *              proper authorization checks allows an attacker to modify arbitrary objects.
+ * @kind problem
+ * @problem.severity error
```

Comment:
Do you think it might be better to rate this as a `warning` rather than an `error`?